pkg/subctl/cmd/deploybroker.go

```diff
@@ -40,6 +44,13 @@ func init() {
 		"Don't install the Submariner dataplane on the broker (default)")
 	// TODO (skitt) make this generic for potentially multiple plugins (see below too)
 	lighthouse.AddFlags(deployBroker, "service-discovery")
+
+	deployBroker.PersistentFlags().BoolVar(&globalnetEnable, "globalnet-enable", false,
+		"Enable support for Overlapping CIDRs in connecting clusters")
+	deployBroker.PersistentFlags().StringVar(&globalnetCidrRange, "globalnet-cidr-range", "169.254.0.0/16",
+		"Global CIDR supernet range for allocating GlobalCIDRs to each cluster")
+	deployBroker.PersistentFlags().UintVar(&globalnetClusterSize, "globalnet-cluster-size", 8192,
```

Comment:
Can you update the commit message as well (it's stale)

i.e., globalnet-cluster-size: Size in bits of each cluster. Default is 8